plugins/source/gandi/go.mod

```diff
@@ -3,20 +3,20 @@ module github.com/cloudquery/cloudquery/plugins/source/gandi
 go 1.19
 
 require (
+	github.com/apache/arrow/go/v13 v13.0.0-20230509040948-de6c3cd2b604
 	github.com/cloudquery/plugin-pb-go v1.0.8
-	github.com/cloudquery/plugin-sdk/v2 v2.7.0
+	github.com/cloudquery/plugin-sdk/v3 v3.6.2
```

Comment:
```suggestion
	github.com/cloudquery/plugin-sdk/v3 v3.6.4
```

Reply:
sorry couldn't keep up with all the sdk releases.